charts/kubewarden-crds/templates/clusterpolicyreports.yaml

```diff
@@ -1,4 +1,3 @@
-{{- if .Values.installPolicyReportCRDs }}
```

Comment:
This looks wrong. I think we need to update our update script. This is caused because updatecli will copy the crds from the audit scanner release. Which does not have this `if`.